include/verilatedos.h

```diff
@@ -163,14 +163,22 @@
 // Comment tag that Function is pure (and thus also VL_MT_SAFE)
 #define VL_PURE
 // Comment tag that function is threadsafe when VL_THREADED
+# if defined(__clang__)
+#define VL_MT_SAFE __attribute__((annotate("MT_SAFE")))
+# else
 #define VL_MT_SAFE
+# endif
```

Comment:
I don't think this does anything right now?  I presume you will be adding a check at some point, I think this is better merged along with that.

Reply:
BTW similar below

Reply:
I'm using this annotate in clang-callgraph to check if function is marked as ``VL_MT_SAFE`` or ``VL_MT_UNSAFE``. ``gcc`` produces warning for ignored attributes, so I'm only setting it for ``clang``.